speech/cloud-client/src/main/java/com/example/speech/InfiniteStreamRecognize.java

```diff
@@ -66,15 +69,31 @@
   private static ByteString tempByteString;
 
   public static void main(String... args) {
+    InfiniteStreamRecognizeOptions options = InfiniteStreamRecognizeOptions.fromFlags(args);
+    if (options == null) {
+      // Could not parse.
+      System.exit(1);
+    }
+
     try {
-      infiniteStreamingRecognize();
+      infiniteStreamingRecognize(options.langCode);
     } catch (Exception e) {
       System.out.println("Exception caught: " + e);
     }
   }
 
+  public static String convertMillisToDate(double milliSeconds, DecimalFormat format) {
```

Comment:
Let's move the `DecimalFormat` setup into this method.

This method is only called once and there is no need to customize the `format`

```java
public static String convertMillisToDate(double milliSeconds) {
    long millis = (long) milliSeconds;
    DecimalFormat format = new DecimalFormat();
    format.setMinimumIntegerDigits(2)
    return String.format("%s:%s /",
            format.format(TimeUnit.MILLISECONDS.toMinutes(millis)),
            format.format(TimeUnit.MILLISECONDS.toSeconds(millis)
                    - TimeUnit.MINUTES.toSeconds(TimeUnit.MILLISECONDS.toMinutes(millis)))
    );
}
```